src/fill_ontologies.py

```diff
@@ -13,200 +14,315 @@
+                        ontologies_dict, known_iri = search_child_term(cell.value, schema_info)  # Search OLS
+                        if zooma:
+                            zooma_ann_dict = search_zooma(cell.value, schema_info)  # Search ZOOMA
+                            if ontologies_dict and zooma_ann_dict:  # Combine OLS and ZOOMA ontologies
+                                ontologies_dict = {**ontologies_dict, **zooma_ann_dict}
+                            elif not ontologies_dict and zooma_ann_dict:
+                                ontologies_dict = zooma_ann_dict
+                            elif not ontologies_dict and not zooma_ann_dict:
+                                term = input("Term '{}' was not found (Property = {}).\nPlease input it manually:".format(cell.value, programmatic_key))
+                                ontologies_dict, known_iri = search_child_term(term, schema_info)
+                        if ontologies_dict:  # Select term from found ontologies
+                            ontology, known_iri = select_term(ontologies_dict, cell.value, programmatic_key,
+                                                              schema_info, zooma, known_iri)
+                            known_terms.append(cell.value)
+                            if isinstance(ontology, list):  # If multi ontology returned, concatenate values
+                                ontologies = {}
+                                ontologies["obo_id"] = "||".join([ontology_element["obo_id"] for ontology_element in ontology if "obo_id" in ontology_element])
+                                ontologies["label"] = "||".join([ontology_label["label"] for ontology_label in ontology if "obo_id" in ontology_label])
+                                ontology = ontologies
+                            known_ontologies[cell.value] = ontology
+                            sheet["{}{}".format(get_column_letter(cell.column + 1), cell.row)].value = known_ontologies[cell.value]["obo_id"]
+                            sheet["{}{}".format(get_column_letter(cell.column + 2), cell.row)].value = known_ontologies[cell.value]["label"]
+        save_workbook(file_path, wb)
+
+    except KeyboardInterrupt:
+        save_workbook(file_path, wb, "_partial_ontologies")
+
+
+def main(args):
+    wb = openpyxl.load_workbook(args.wb_path)
+    script_path = os.path.realpath(__file__)
+    script_dir = os.path.split(script_path)[0]
+    pickled_schema_path = os.path.join(script_dir, 'pickled_schemas.pkl')
+    try:
+        with open(pickled_schema_path, 'rb') as pickled_schemas:
+            json_schemas = pickle.load(pickled_schemas)
+    except FileNotFoundError:
+        schema = SchemaTemplate(ingest_api_url="http://api.ingest.staging.archive.data.humancellatlas.org")
+        with open(pickled_schema_path, 'wb') as output:
+            pickle.dump(schema.json_schemas, output)
+        json_schemas = schema.json_schemas
```

Comment:
```suggestion
        schema = SchemaTemplate(ingest_api_url="http://api.ingest.staging.archive.data.humancellatlas.org")
        json_schemas = schema.json_schemas
        with open(pickled_schema_path, 'wb') as output:
            pickle.dump(json_schemas, output)
```

Just a small rearrangement so that schemas.json_schemas is not accessed twice (not really important but I think this is clearer)